hs-abci-sdk/src/Tendermint/SDK/BaseApp/Metrics/Prometheus.hs

```diff
@@ -205,7 +205,7 @@ evalMetrics state@MetricsState{..} = do
       start <- liftIO $ getCurrentTime
       a <- runT action
       end <- liftIO $ getCurrentTime
-      let time = fromRational . (* 1000.0) . toRational $ (end `diffUTCTime` start)
+      let time = realToFrac (end `diffUTCTime` start)
```

Comment:
suggestion: use `ms` as your unit -- by the looks of your graphs your ops are in that range and you're just giving yourself floating imprecision for no good reason by choosing seconds